cli/src/commands/create/emails.rs

```diff
@@ -36,9 +39,17 @@ pub struct CreateEmailsArgs {
     #[structopt(short = "n", long = "no-charge")]
     /// Whether to attempt to bypass billing (internal only)
     no_charge: bool,
+
+    #[structopt(short = "y", long = "yes")]
+    /// Consent to ai unit charge. Suppresses confirmation prompt.
+    consent_to_ai_unit_charge: bool,
```

Comment:
I would say it's fine to just name this argument `yes`, as in the user has opted to skip all confirmation prompts.

Would be nice for adding future prompts